### DISCUSSION TITLE
퀘스트 완료시 앱에서 사용가능한 재화 제공

### DISCUSSION THREAD

**Discussion by xowjd913** [naverz/zepeto-studio-kor#972] [5. Q&A]
게임을 만들고 있는데
게임의 퀘스트를 전부 완료하면 제패토 앱에서 사용가능한 재화(코인, 다이아)를 줄까합니다.

혹시 해당 기능에 대한 API가 만들어져있을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 제페토 앱의 공식 재화는 어뷰징 및 악용의 우려가 있으므로 
크리에이터 월드에서 지급을 하지 않는 것이 현재 정책이므로 해당 기능을 사용하실 수 없습니다.
양해 부탁 드립니다.
감사합니다.
